### ISSUE TITLE
Issue with parsing 7-Zip version if alpha version is installed

### ISSUE THREAD

**Issue by gregoral** [microsoft/navcontainerhelper#1913]
**Expand-7zipArchive** helper function has an issue with parsing 7-Zip version information.
The function does not fail, but uses PowerShell built-in Expand-Archive commandlet instead.

This is not ideal, because Expand-Archive has an issue with paths over 260 charaters long.
That can lead to a different issue / error, which I will file separately.

The issue is triggered on systems where an unreleased version of 7-Zip is installed.
Tested with 7-Zip version: **"21.01 alpha"**.

The problem is with this line of code:
```
$use7zip = [decimal]::Parse([System.Diagnostics.FileVersionInfo]::GetVersionInfo($7zipPath).FileVersion, [System.Globalization.CultureInfo]::InvariantCulture) -ge 19
```

The exception thrown is this one:
Exception calling "Parse" with "2" argument(s): "Input string was not in a correct format."

Version checking code was added as a solution for this issue: #1138

**Proposed solution**
Solution is to avoid parsing the file version information and use FileMajorPart property instead.
This line of code fixes the issue:
```
$use7zip = [System.Diagnostics.FileVersionInfo]::GetVersionInfo($7zipPath).FileMajorPart -ge 19
```

Pull request with a fix for this issue: #1914

**Comment by freddydk** [comment]
Thanks for the PR - I have merged it.

**Comment by gregoral** [comment]
I'm glad I was able to make a small contribution to an otherwise great tool.

**Comment by freddydk** [comment]
Shipped in 2.0.12
